resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -316,6 +316,14 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val AUTO_STOP_ACTIVE_SPARK_CONTEXTS =
+    ConfigBuilder("spark.kubernetes.submit.autoStopActiveSparkContexts")
+      .version("3.5.0")
+      .doc("When set to true, on Kubernetes Spark will stop all the active Spark contexts after " +
```

Comment:
What about phrasing it as: `When set to true, Spark on Kubernetes will stop all active Spark contexts once non-shell application' main methods are finished` instead?